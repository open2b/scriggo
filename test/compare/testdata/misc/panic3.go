// skip

// paniccheck

package main

func main() {
	defer func() {
		recover()
		panic(2)
	}()
	panic(1)
}
