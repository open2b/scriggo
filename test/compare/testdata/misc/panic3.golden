panic: 1 [recovered]
		panic: 2